logs/models.go

```diff
@@ -0,0 +1,17 @@
+// This package is for logging runtime data.
+package logs
+
+import (
+	"github.com/sirupsen/logrus"
+)
+
+type Log struct{ // TODO consider optimizations with enum like structure to give options to log creator
```

Comment:
Noted that for MVP we want to keep things as simple as possible, but design wise this is tightly coupling the Logger implementation throughout the rest of the modules.